pkg/broker/ipsec_psk.go

```diff
@@ -40,3 +41,25 @@ func NewBrokerPSKSecret(bytes int) (*v1.Secret, error) {
 func GetIPSECPSKSecret(clientSet clientset.Interface, brokerNamespace string) (*v1.Secret, error) {
 	return clientSet.CoreV1().Secrets(brokerNamespace).Get(ipsecPSKSecretName, metav1.GetOptions{})
 }
+
+func GetIPSECPSKBytes(clientSet clientset.Interface, brokerNamespace string) ([]byte, error) {
+	secret, err := GetIPSECPSKSecret(clientSet, brokerNamespace)
+	if err != nil {
+		return nil, err
+	}
+
+	// FIXME: Use shared var for PSK byte array length everywhere
+	psk := make([]byte, 48)
+	err = secret.Unmarshal(psk)
```

Comment:
there must be a psk.Data["psk"] structure inside that secret with the data already in bytes, I think. I will check it.